MantidPlot/src/ProjectRecoveryGUIs/ProjectRecoveryModel.h

```diff
@@ -35,7 +35,6 @@ class ProjectRecoveryModel {
   void fillFirstRow();
   void fillRow(const Poco::Path &path, const std::string &checkpointName);
   void updateCheckpointTried(const std::string &checkpointName);
-  bool checkRecoverWasASuccess(const std::string &projectFile);
```

Comment:
Can you help me understand the reason for touching the `MantidPlot` recovery functionality?

Reply:
This was never actually used so it was removed. It's fair enough that you would be dubious of this.